tests/test_utils.py

```diff
@@ -97,31 +97,19 @@ def test_typing():
     assert 'OptionType' in arku.typing.__all__
 
 
-def test_redis_settings_validation():
-    class Settings(BaseModel):
-        redis_settings: RedisSettings
+def test_import_module():
+    assert arku.utils.import_string('os.path') == os.path
 
-        @validator('redis_settings', always=True, pre=True)
-        def parse_redis_settings(cls, v):
-            if isinstance(v, str):
-                return RedisSettings.from_dsn(v)
-            else:
-                return v
 
-    s1 = Settings(redis_settings='redis://foobar:123/4')
```

Comment:
Why the tests are gone?

Reply:
Modified this test to pass without Pydantic